src/DFTK.jl

```diff
@@ -55,10 +55,10 @@ export compute_fft_size
 export G_vectors, G_vectors_cart, r_vectors, r_vectors_cart
 export Gplusk_vectors, Gplusk_vectors_cart
 export Kpoint
-export G_to_r
-export G_to_r!
-export r_to_G
-export r_to_G!
+export ifft
+export ifft!
+export fft
+export fft!
```

Comment:
Should be no need as this is now handled by AbstractFFTs.

Reply:
OTOH we probably want people to have access to these functions even though they're not `using AbstractFFTs`, so ok, keep as is!

Reply:
Yes, otherwise `fft` will actually be a different function from `AbstractFFTs.fft`, which just happens to have the same name.